toolchain/check/handle_operator.cpp

```diff
@@ -262,18 +281,14 @@ auto HandlePrefixOperatorNot(Context& context, Parse::NodeId parse_node)
   return true;
 }
 
-auto HandlePrefixOperatorPlus(Context& context, Parse::NodeId parse_node)
-    -> bool {
-  return context.TODO(parse_node, "HandlePrefixOperatorPlus");
-}
```

Comment:
Maybe worth mentioning in the PR description that this already prevents us from leaving stale handlers around when removing parse nodes, and citing this one as an example?

Reply:
Done.